src/main/java/org/broadinstitute/hellbender/tools/walkers/sv/PairedEndAndSplitReadEvidenceCollection.java

```diff
@@ -434,4 +512,165 @@ public int compare(final DiscordantRead o1, final DiscordantRead o2) {
             return internalComparator.compare(o1, o2);
         }
     }
+
+    @VisibleForTesting
+    final static class AlleleCounter {
```

Comment:
Yes.  That's designed to be used by a LocusWalker, but we need to be a ReadWalker to gather the other types of evidence.  It would be lots of extra code to generate ReadPileups.